docs/sources/write/links/index.md

```diff
@@ -0,0 +1,98 @@
+
+- [Hugo `relref` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#relref)
+
+## Link from source content that's used in multiple projects
+
+Use the `docs/reference` shortcode.
+
+The source is reused as described in [Reuse directories of content with Hugo mounts](https://grafana.com/docs/writers-toolkit/write/reuse-content/reuse-directories/).
+For more information and examples, refer to [`docs/reference` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsreference).
+
+## Link to grafana.com pages
+
+Use a fully qualified URL with version substitution syntax (if needed).
+Version substitution is necessary for fully qualified URLs to link to the correct version of documentation.
+Usually, this is the current version of documentation.
+
+In versioned documentation, ensure that you set the appropriate version in the root `_index.md` file for your documentation.
+For example, the following YAML, merged with the existing front matter in the root `_index.md` file sets `GRAFANA_VERSION` to be `latest` for that page and all child pages.
+
+```yaml
+cascade:
+  GRAFANA_VERSION: latest
+```
+
+### Examples
+
+**Link to Grafana documentation**:
+
+Start with `https://grafana.com/docs/grafana/<GRAFANA_VERSION>/`, and add the rest of the URL path.
+
+For example, to link to the [Developers](https://grafana.com/docs/grafana/latest/developers) page with version substitution,
+use `https://grafana.com/docs/grafana/<GRAFANA_VERSION>/developers/`.
+
+- If you're linking from Grafana documentation, `<GRAFANA_VERSION>` is substituted with the version inferred from the page's URL.
+- If you're linking from other documentation, `<GRAFANA_VERSION>` is substituted with the value of `GRAFANA_VERSION` set in the page's front matter.
+
+**Link to Grafana Cloud documentation**:
+
+Grafana Cloud documentation is not versioned so no version substitution syntax is needed.
+Use the fully qualified URL.
+For example, to link to the [Author and run tests](https://grafana.com/docs/grafana-cloud/k6/author-run/) page, use `https://grafana.com/docs/grafana-cloud/k6/author-run/`.
+
+**Link to Mimir documentation**:
+
+Start with `https://grafana.com/docs/grafana/<MIMIR_VERSION>/`, and add the rest of the URL path.
+
+For example, to link to the [Release notes](https://grafana.com/docs/mimir/latest/release-notes/) page with version substitution,
+use `https://grafana.com/docs/mimir/<MIMIR_VERSION>/release-notes/`.
+
+- If you're linking from Mimir documentation, `<MIMIR_VERSION>` is substituted with the version inferred from the page's URL.
+- If you're linking from other documentation, `<MIMIR_VERSION>` is substituted with the value of `MIMIR_VERSION` set in the page's front matter.
+
+## Link to external pages
+
+Use the fully qualified URL.
+For example, `https://github.com`.
+
+## Link to page headings
+
+You can optionally include an anchor to link to a specific heading in the destination page.
```

Comment:
Wanna speak to two use cases here: linking within a page or linking from another page to a heading on a page. Thinking something like.
````suggestion
Link to a heading on a page in one of two ways.

From within the same page:

```markdown
Read more in the [Configuration section](#configuration) of this page.
```

From a different page:

```markdown
Read more in the [Grafana Open Source section of the Introduction page](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/fundamentals/#grafana-open-source
```
````